index.Dockerfile

```diff
@@ -5,6 +5,7 @@ ARG BUNDLE_IMGS=quay.io/ocs-dev/odf-operator-bundle:latest
 RUN opm index add \
 --bundles "${BUNDLE_IMGS}" \
 --out-dockerfile index.Dockerfile \
+--permissive \
```

Comment:
This can be squashed into other commit where this file was added

Reply:
Valid.